lamagraph-compiler/test/Lamagraph/Compiler/ParserGoldenTest.hs

```diff
@@ -0,0 +1,41 @@
+module Lamagraph.Compiler.ParserGoldenTest (parserGoldenTestsAST) where
+
+import Relude
+
+import Prettyprinter
+import Prettyprinter.Render.Text
+import System.FilePath
+import Test.Tasty
+import Test.Tasty.Golden
+
+import Lamagraph.Compiler.Parser
+import Lamagraph.Compiler.PrettyAST ()
+
+render :: Doc ann -> LText
+render = renderLazy . layoutPretty defaultLayoutOptions
+
+parserGoldenTestsAST :: IO TestTree
+parserGoldenTestsAST = do
+  lmlFiles <- findByExtension [".lml"] "test/parserGolden/source"
+  return $
+    testGroup
+      "Golden tests AST"
+      [ goldenVsString (takeBaseName lmlFile) resLmlFile (helper lmlFile)
+      | lmlFile <- lmlFiles
+      , let resLmlFile = addExtension (changePath lmlFile) "ast"
```

Comment:
Can `"ast"` be moved to the top-level binding and reused? It used in this file twice.